events-api/src/shared/validators/cardValidators/card-expiry-date.validator.ts

```diff
@@ -0,0 +1,32 @@
+import { ValidatorConstraint, ValidatorConstraintInterface, ValidationArguments } from 'class-validator';
+import dayjs from 'dayjs';
+
+@ValidatorConstraint({ name: 'cardExpiry', async: false })
```

Comment:
Better to use a function rather than a class. It is more readable and more common approach

Reply:
Done